eq-author/src/components/CommentsPanel/index.test.js

```diff
@@ -0,0 +1,221 @@
+import React from "react";
+import { render, flushPromises, fireEvent } from "tests/utils/rtl";
+import actSilenceWarning from "tests/utils/actSilenceWarning";
+
+import CommentsPanel from "./";
+import COMMENT_QUERY from "./commentsQuery.graphql";
+import COMMENT_ADD from "./createNewComment.graphql";
+
+describe("Comments Pane", () => {
+  let props, pageId, queryWasCalled, mocks;
+
+  // this is just a little hack to silence a warning that we'll get until we
+  // upgrade to 16.9: https://github.com/facebook/react/pull/14853
+  // https://github.com/testing-library/react-testing-library#suppressing-unnecessary-warnings-on-react-dom-168
+  actSilenceWarning();
+
+  beforeEach(() => {
+    queryWasCalled = false;
+    props = {
```

Comment:
No need for this props object and no need to pass the props into any of the components below.